### PR TITLE
Update the README KAFKA create topic for issue #221.

### PR DESCRIPTION
**Pull request by afonsoaugusto**
Signed-off-by: Afonso Rodrigues <afonso.rodrigues@maxmilhas.com.br>

# Pull Request

## What I did
Change README with execution the command `rit create topic`

## How I did it
Change README

## How to verify it
Check this PR
Alt Text

## What kind of change does this PR make

- [ ] Major
- [ ] Minor
- [X] Patch

## Description for the changelog

 issue #221

### CODE REVIEW

**Review by fabianofernandeszup — APPROVED** [review]

**Review by fabianofernandeszup — CHANGES_REQUESTED** [review]
First of all, thanks for the contribution.
There was a conflict in your PR. I already did the initial analysis and everything seems fine to me.
As soon as I resolve the conflict, I can accept it.

**Comment by GuillaumeFalourd** [comment]
Hasn't this Issue (update the rit kafka create topic readme) already been resolved by the #309 PR ?

**Review by fabianofernandeszup — DISMISSED** [review]

**Comment by afonsoaugusto** [comment]
Hi @fabianofernandeszup , conflicts is solved.

File: kafka/create/topic/README.md [hunk 1]
```diff
@@ -1,24 +1,58 @@
...
+<!-- markdownlint-disable-file MD013 -->
+<!-- markdownlint-disable-file MD033 -->
+
+# create kafka topic
+
+## Premisses
+
+- The kafka cluster is accessible
 
 ## Premisse
```
Comment (GuillaumeFalourd) [inline kafka/create/topic/README.md:10]: Hi @afonsoaugusto , there are 2 Premisse topic, we could remove one.

File: kafka/create/topic/README.md [hunk 2]
```diff
@@ -1,24 +1,58 @@
...
+
+```bash
+echo '{"urls":"localhost:9092", "name":"topic-stdin", "replication":"1", "partitions":"1"}' | rit kafka create topic --stdin --docker
+```
+
 ## Description
 
-This formula create a topic on the inform kafka URL.
-You can test it using docker with the `rit docker generate compose` formula.
+This create kafka topic command allows the user to delete a bucket in AWS S3
```
Comment (GuillaumeFalourd) [inline kafka/create/topic/README.md:42]: I think there is a problem with the description talking about AWS S3 bucket here.
You could maybe re-use the command above :

```
This formula create a topic on the inform kafka URL.
You can test it using docker with the `rit docker generate compose` formula.
```

**Review by GuillaumeFalourd — CHANGES_REQUESTED** [review]

File: kafka/create/topic/README.md (hunk 2 above)
Comment (afonsoaugusto) [reply kafka/create/topic/README.md:42]: Hi @GuillaumeFalourd , My apologies the bad description in my commit. I kept a template with addiction the another implementation. I will update with this description.

**Review by afonsoaugusto — COMMENTED** [review]
Update the README.md with orientations

**Review by GuillaumeFalourd — APPROVED** [review]

**Comment by GuillaumeFalourd** [comment]
You just need to sign-off the DCO @afonsoaugusto and we'll merge your contribution 😄 
Thanks for the update 👍🏼

**Comment by afonsoaugusto** [comment]
@GuillaumeFalourd ,
How to sign-off the commit after change?

**Review by GuillaumeFalourd — APPROVED** [review]

**Pull request merged**